src/schematools/types.py

```diff
@@ -1567,6 +1574,7 @@ def subfields(self) -> list[DatasetFieldSchema]:
 
         If self is not an object or array, the return value is an empty iterator.
         """
+
```

Comment:
I don't mind this, but do black and flake8 like it? I thought one of them warns about space after docstring.